Examples/Algorithms/TrackFitting/src/TrackFittingAlgorithm.cpp

```diff
@@ -86,7 +86,7 @@ ActsExamples::ProcessCode ActsExamples::TrackFittingAlgorithm::execute(
   std::vector<const Acts::Surface*> surfSequence;
   for (std::size_t itrack = 0; itrack < protoTracks.size(); ++itrack) {
     // Check if you are not in picking mode
-    if (m_cfg.pickTrack > 0 and m_cfg.pickTrack != static_cast<int>(itrack)) {
+    if (m_cfg.pickTrack > -1 and m_cfg.pickTrack != static_cast<int>(itrack)) {
```

Comment:
personally I prefer `!= -1` in such a case becuase `-1` encodes "do not pick" but with `>` every negative int will do the same

Reply:
As far as I have seen, we have not yet had a similar special case handling in Acts.
I think excluding all negative ints makes sense here.